charts/spire/charts/spire-server/values.yaml

```diff
@@ -351,11 +347,59 @@ tornjak:
     successThreshold: 1
     timeoutSeconds: 5
 
+  # tornjak - Tornjak specific configuration
   config:
-    # -- persistent DB for storing Tornjak specific information
+    # -- Persistent DB for storing Tornjak specific information
     dataStore:
       driver: "sqlite3"
       file: "/run/spire/data/tornjak.sqlite3"
+
+    # Tornjak HTTP service
+    http:
+      # -- Enables Tornjak HTTP (insecure) service
+      enabled: true
+      # -- Container port value for HTTP
+      port: 10000
+      # -- Service to handle Tornjak HTTP connection
+      service:
+        type: ClusterIP
+        port: 10000
+        annotations: {}
+
+    # Tornjak TLS service
+    tls:
+      # -- Enables Tornjak TLS service
+      enabled: false
+      # -- Container port value for TLS
+      port: 20000
+      # -- Certificate for TLS connection
+      cert: "/opt/spire/sample-keys/cert.pem"
```

Comment:
Does cert and key filename inside the container need to be configurable? k8s has standardized secrets of type tls.

Reply:
I was initially thinking we could configure multiple keys e.g. different keys for TLS and mTLS but that's probably overkill here. I am fine with hardcoding the path to the key and cert